extensions/rocksdb-repos/DatabaseContentRepository.h

```diff
@@ -64,7 +64,6 @@ class StringAppender : public rocksdb::AssociativeMergeOperator {
   }
 
  private:
-
```

Comment:
the previous two lines could be removed, too

Reply:
That is a different error of a different scope than the errors this PR meant to fix.